frontend/src/metabase/parameters/components/ParameterValueWidget.jsx

```diff
@@ -33,7 +33,7 @@ import {
 } from "metabase-lib/parameters/utils/parameter-type";
 
 import ParameterFieldWidget from "./widgets/ParameterFieldWidget/ParameterFieldWidget";
-import S from "./ParameterWidget.css";
+import S from "./ParameterValueWidget.css";
```

Comment:
Rename the style file since most of the style rules are actually used by `ParameterValueWidget`